perf/benchmark/values.yaml

```diff
@@ -15,6 +15,7 @@ appresources1:
 
 fortioImage: fortio/fortio:latest
```

Comment:
@mandarjog not relevant for this review, however, you may wish to pin to specific versions of these benchmarking tools to improve reproducibility...

Reply:
@sdake we recently did a small change in fortio/fortio repo, and pushed that image to istio-testing container registry.